### PR TITLE
Logic adding

### PR DESCRIPTION
**Pull request by lisandroseia**
In this branch I added:
2 modules inside the [logic folder](https://github.com/lisandroseia/math-magicians/tree/logic-adding/src/logic) with 2 functions to make the calculator work
Used those functions inside my [calculator class](https://github.com/lisandroseia/math-magicians/blob/logic-adding/src/components/Calculator.js)
Installed the 'Big dependancie'

### CODE REVIEW

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    return {};
+  }
+
+  // Button must be an operation
+
+  // When the user presses an operation button without having entered
+  // a number first, do nothing.
+  // if (!obj.next && !obj.total) {
+  //   return {};
+  // }
```
Comment (Cathella) [inline src/logic/calculate.js]: Please kindly remove these chunks of code to make your code more readable

File: src/logic/calculate.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+/**
+ * Given a button name and a calculator data object, return an updated
+ * calculator data object.
+ *
+ * Calculator data object contains:
+ *   total:s      the running total
+ *   next:String       the next number to be operated on with the total
+ *   operation:String  +, -, etc.
+ */
+export default function calculate(obj, buttonName) {
```
Comment (Cathella) [inline src/logic/calculate.js]: Hello @lisandroseia, Please kindly remove these chunks of code to make your code more readable

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Required 🛑

### Hi @lisandroseia 👋🏻, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## To Highlight 👏🏻

✅ No linter errors.
✅ You used correct Github Flow.
✅ Implemented the click event for all buttons

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/logic/calculate.js (hunk 2 above)
Comment (lisandroseia) [reply src/logic/calculate.js]: Hi @Cathella, thanks for the feedback, I will implement the changes right now

**Review by julie-ify — APPROVED** [review]
## Hi @lisandroseia, 

# Status: Approved ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**